CMakeLists.txt

```diff
@@ -8,10 +8,12 @@ daq_setup_environment()
 find_package(appfwk REQUIRED)
 find_package(dfmessages REQUIRED)
 
-daq_add_plugin(TriggerDecisionEmulator duneDAQModule LINK_LIBRARIES appfwk::appfwk dfmessages::dfmessages SCHEMA)
+daq_add_library(TimestampEstimator.cpp LINK_LIBRARIES appfwk::appfwk dfmessages::dfmessages)
```

Comment:
Is the .cpp supposed to be here?

Reply:
Yes, I think so. The `TimestampEstimator` class is used by the `TriggerDecisionEmulator` plugin

Reply:
OK, got it...`daq_add_library` isn't set up like a standard CMake `add_library` where the first argument would be the target name...